apps/examples/err_report_demo/err_report_main.c

```diff
@@ -295,17 +295,35 @@ void error_report_process(int argc, char *argv[])
 		printf("%s", ERR_REPORT_USAGE);
 		return;
 	}
-	if (strcmp(argv[2], "send") == 0) {
+	if (strcmp(argv[2], "test") == 0) {
 		if (argc < 4) {
-			error_report_records(NULL);
+			error_report_test(NULL);
 		} else {
-			error_report_records(argv[3]);
+			error_report_test(argv[3]);
 		}
-	} else if (strcmp(argv[2], "test") == 0) {
+	} else if (strcmp(argv[2], "report_send") == 0) {
 		if (argc < 4) {
-			error_report_test(NULL);
+			error_report_records(NULL);
 		} else {
-			error_report_test(argv[3]);
+			error_report_records(argv[3]);
+		}
+	} else if (strcmp(argv[2], "report_print")) {
```

Comment:
In my opinion, list of commands might be 'test, send, print'. Please let me know if you any reason to attach prefix 'report_'?

Reply:
Thanks for the comment. Yes, the prefix is required to distinguish between printing an error report and printing trace calls (for callgraph) comprising different function invocations. In general, for each feature, a generic print method could be applied with a parameterized argument. This is however, beyond the scopr of this pull request. So for now, I mention report_print to make the context explicit.